pkg/sass_api/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 12.0.2
+
+* `unifyComplex()` and `unifyCompound()` now preserve the original order of
+  pseudo-classes and pseudo-elements within individual selectors when combining
+  them.
```

Comment:
Since this isn't an API that's specific to `sass_api`, just put "No user-visible changes" here.